_clubs/naak/naak-earthcorps-projects-for-2019.md

```diff
@@ -0,0 +1,65 @@
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Yellow archangel, Purple loosestrife, Tansy ragwort and Garlic mustard.
+
+#### Reforestation of Yellow Lake NGPA with 1 gallon native tree stock
+Add native trees including western red cedar and western hemlock to specific areas for reforestation in the fall of the year.  TBD based upon need and sustainability.
+
+#### Mountain View NGPA (including Scotch broom behind pickleball court)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Tansy ragwort and English holly.
+
+#### Klahanie Perimeter NGPA
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Tansy ragwort and Yellow Archangel.  All control work will be based upon potential impacts of the Issaquah/Fall City road expansion.
+
+#### Pea Patch Area (NGPA)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries and Tansy ragwort.
+
+#### Sutter Pond, Gander Pond, Lake Park, Heatherwood, 242<sup>nd</sup> wetland, 242<sup>nd</sup> Park and Native Plant Demonstration Garden (multiple small natural areas)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Tansy ragwort, Japanese knotweed and Purple loosestrife.  Increase level of compost in Native Plant Demonstration Garden and add native plants, as needed.  Water plantings from 2018 in Lake Park, as needed.
+
+#### Gas Pipeline Trail Natural Area (north and south sections)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Yellow archangel, and Tansy ragwort.
+
+#### Oxford Park NGPA
+Continue control of noxious weeds such as English ivy, Scotch broom and Himalayan blackberries.
+
+#### Highland Gardens NGPA
+Continue control of noxious weeds such as English ivy, Scotch broom and Himalayan blackberries.  
+
+#### Queens Bog NGPA
+Continue control of noxious weeds such as English ivy, Scotch broom, Yellow archangel and Himalayan blackberries.  
+
+#### Power Line Trail Natural Area (including adjacent to ballfield)
+Continue control of noxious weeds such as English ivy, Scotch broom, Tansy ragwort and Himalayan blackberries.  
+
+#### Skye Landing
+Continue control of noxious weeds such as English ivy, Scotch broom, Tansy ragwort and Himalayan blackberries.  Add material for erosion control of hillside and add plants for stabilization, as needed.
+
+#### Native Plant Pollinator Habitat
+Add a Native Plant Pollinator Habitat, behind the Native Plant Demonstration Garden.  This includes coordination with Bonneville Power Administration, site clearing and preparation, and the addition of appropriate native plants.
+
+#### Community Work Parties
+Hold volunteer work parties to control of noxious weeds and improve wildlife habitats, as needed.
+
+#### Wood Duck Nesting Boxes
+Maintain Wood Duck boxes at Gander Pond.

```

Comment:
Merriam Webster does not capitalize "wood duck," but Audubon does. I marked it lowercase in a different section.